dashboard/client/src/pages/dashboard/node-info/NodeInfo.tsx

```diff
@@ -209,9 +209,11 @@ const NodeInfo: React.FC<{}> = () => {
   // Show GPU features only if there is at least one GPU in cluster.
   const showGPUs =
     nodes.map((n) => n.gpus).filter((gpus) => gpus.length !== 0).length !== 0;
+  // Don't show disk on K8s. K8s node disk usage should be monitored elsewhere.
+  const showDisk = !("KUBERNETES_SERVICE_HOST" in process.env);
```

Comment:
this won't work because process.env is hard coded at dashboard build time, and this code will be ran in user's browser... in order to do this you have to make this variable in the dashboard backend (see reporting agent, @kathryn-zhou can help as well)

Reply:
Thanks, that makes sense -- so basically my question is how do I get the variable `IN_KUBERNETES_POD` in `reporter_agent.py` to `NodeInfo.tsx`?
https://github.com/ray-project/ray/blob/master/dashboard/modules/reporter/reporter_agent.py#L29

-- what's the code path?

Reply:
@edoakes do you know how this works?

Reply:
@kathryn-zhou you have added metrics to this reporter agent and pop it up in the frontend before. Can you leave some pointers here?

Reply:
I think what you want to do is have the environment variable check here and report a special number that you use to not display usage on the dashboard:
https://github.com/ray-project/ray/blob/master/dashboard/modules/reporter/reporter_agent.py#L238

Reply:
e.g., if we report `0` used and `0` available we don't even render it

Reply:
sounds reasonable, will try that

Reply:
I haven't added metrics to the frontend before, but @DmitriGekhtman it might be worthwhile to look at how GPU metrics are shown on the dashboard. I believe they are only shown if there is a GPU in the cluster.